app/styles/_tables.less

```diff
@@ -143,6 +143,19 @@
         }
       }
     }
+    // remove extra 2px border on <tr> in nested ng-repeats.
+    // since we cannot select the "first element of parent with class ..."
+    // select every <tr> with the .nested-loop class, apply a 0px border-top-width,
+    // and then re-apply the 2px border-top-width to every tr.nested-loop child
+    // except the first one to avoid tables with 4px border-top-widths.
+    > tbody.nested-loop-container {
+      > tr.nested-loop {
+        border-top-width: 0;
+      }
+      > tr.nested-loop ~ tr.nested-loop {
+          border-top-width: 2px;
```

Comment:
@spadgett 

The border comes from the first non-hidden `<tr>` (with className  `nested-loop`) (see attached image), but normally, this first non-hidden `<tr>` would not have a `border-top-width` due to the css rule `tbody > tr + tr {...` which would only apply the border to `<tr>` tags that immediately follow other `<tr>` tags. Normally the top `2px` border that is seen is the overall table's top-border. Because the hidden `<tr>` right above the one in the image still exists in the dom, it forces this first visible `<tr>` to have a `border-top-width`, which combined with the table's existing `2px` results in the top border with extra width that we are currently seeing in these tables.

To get around this, I select all non-hidden `<tr>` elements by adding the class `nested-loop` to them, remove their `border-top-width`, and then re-apply it to every one except the first `.nested-loop` tr. 

![service-page-first-element-w-class](https://cloud.githubusercontent.com/assets/3902875/21354521/7306ed92-c698-11e6-8a0f-1ea38e7ae737.png)